overlord/hookstate/ctlcmd/helpers.go

```diff
@@ -69,6 +69,18 @@ func getServiceInfos(st *state.State, snapName string, serviceNames []string) ([
 
 var servicechangeImpl = servicestate.Change
 
+func queueCommand(context *hookstate.Context, ts *state.TaskSet) {
+	// queue command task after all existing tasks of the hook's change
+	st := context.State()
+	st.Lock()
+	defer st.Unlock()
+
+	change := context.Task().Change()
```

Comment:
And if this is called with a context which is not a hook? BAM!

I never won a bet so quickly. ;-)